process/project_proposals.adoc

```diff
@@ -36,27 +36,21 @@ image::incubation-process.png[Incubation process]
-. *SIG Assessment* _1-2 months_
-   * Project presents to SIG
-   * Lightweight - this is pre-DD
-   * Confirms that project accepts the https://github.com/cncf/toc/blob/main/process/graduation_criteria.adoc#incubation-stage[Incubation requirements]
-   * SIG makes recommendation to TOC: Sandbox / Incubation / Decline 
-   * Note that this step is still meant to be a lightweight, pre-DD process
-. *TOC Incubation Sponsor*
-   * If a TOC member steps forward as an Incubation Sponsor, due diligence commences
+. *TOC Incubation Sponsor* 
+   * The project cannot move forward to Due Diligence unless and until a TOC member steps forward as an Incubation Sponsor. 
+   * The TOC may agree that the project does not (yet) meet the https://github.com/cncf/toc/blob/main/process/graduation_criteria.adoc#incubation-stage[Incubation requirements] and suggest that project apply for Sandbox instead. 
+   * If a TOC Incubation Sponsor has not stepped forward within two months after the proposal PR is submitted, projects may request that their project proposal is discussed at a forthcoming TOC meeting by adding it to the Working Doc. The outcome of this is discussion is either that a sponsor steps forward, or that the TOC votes to admit the project to Sandbox, or the proposal is rejected (projects may reapply after six months). The default outcome is that if a sponsor does not step forward, the proposal is rejected. 
 . *Due Diligence* _2-3 months_
+   * TOC Incubation Sponsor confirms that project meets the https://github.com/cncf/toc/blob/main/process/graduation_criteria.adoc#incubation-stage[Incubation requirements]
    * TOC Incubation Sponsor drives due diligence (see the https://github.com/cncf/toc/blob/main/process/dd-review-template.md[template] and https://github.com/cncf/toc/blob/main/process/due-diligence-guidelines.md[guidelines]).
+   * Typically the DD will involve a presentation to a CNCF SIG, as identified by the TOC Sponsor. This step may be omitted if the TOC Sponsor feels there are readily-available and suitable presentations on video, and that scheduling the presentation to the SIG would delay the process by more than two months. (We do not want unnecessary levels of process or bureaucracy to delay a widely-known and adopted project from joining the CNCF). 
    * TOC Incubation Sponsor can delegate DD work to CNCF SIGs and/or other TOC members.
-   * TOC Incubation Sponsor can ask project maintainers to complete the DD template as a first pass.
+   * TOC Incubation Sponsor can ask project maintainers to complete the DD template. Project maintainers are welcome to start on this in advance of the official DD process, or even in advance of the initial proposal as it may help them ensure they meet all the requirements. The TOC Incubation Sponsor should carefully review and ask questions about the DD as prepared by the project maintainers. 
    * CNCF staff do governance and legal DD.
```

Comment:
Should we elaborate on what this entails, and who would be the point of communications for the project owner?